pkg/scheduler/framework/plugins/volumerestrictions/volume_restrictions_test.go

```diff
@@ -478,6 +480,188 @@ func TestAccessModeConflicts(t *testing.T) {
+	podISCSIDiskNoConflicts := st.MakePod().Volume(ISCSIDiskVolState2).Obj()
+
+	testcases := map[string]struct {
+		pod            *v1.Pod
+		oldObj, newObj interface{}
+		existingPods   []*v1.Pod
+		existingPVC    *v1.PersistentVolumeClaim
+		expectedHint   framework.QueueingHint
+		expectedErr    bool
+	}{
+		"queue-new-object-gcedisk-conflict": {
+			pod:          podGCEDisk,
+			oldObj:       podGCEDiskConflicts,
+			existingPods: []*v1.Pod{},
+			existingPVC:  &v1.PersistentVolumeClaim{},
+			expectedHint: framework.Queue,
+			expectedErr:  false,
+		},
+		"skip-new-object-gcedisk-no-conflict": {
+			pod:          podGCEDisk,
+			oldObj:       podGCEDiskNoConflicts,
+			existingPods: []*v1.Pod{},
+			existingPVC:  &v1.PersistentVolumeClaim{},
+			expectedHint: framework.QueueSkip,
+			expectedErr:  false,
+		},
+		"queue-new-object-awsdisk-conflict": {
+			pod:          podAWSDisk,
+			oldObj:       podAWSDiskConflicts,
+			existingPods: []*v1.Pod{},
+			existingPVC:  &v1.PersistentVolumeClaim{},
+			expectedHint: framework.Queue,
+			expectedErr:  false,
+		},
+		"skip-new-object-awsdisk-no-conflict": {
+			pod:          podAWSDisk,
+			oldObj:       podAWSDiskNoConflicts,
+			existingPods: []*v1.Pod{},
+			existingPVC:  &v1.PersistentVolumeClaim{},
+			expectedHint: framework.QueueSkip,
+			expectedErr:  false,
+		},
+		"queue-new-object-rbddisk-conflict": {
+			pod:          podRBDDiskDisk,
+			oldObj:       podRBDDiskDiskConflicts,
+			existingPods: []*v1.Pod{},
+			existingPVC:  &v1.PersistentVolumeClaim{},
+			expectedHint: framework.Queue,
+			expectedErr:  false,
+		},
+		"skip-new-object-rbddisk-no-conflict": {
+			pod:          podRBDDiskDisk,
+			oldObj:       podRBDDiskNoConflicts,
+			existingPods: []*v1.Pod{},
+			existingPVC:  &v1.PersistentVolumeClaim{},
+			expectedHint: framework.QueueSkip,
+			expectedErr:  false,
+		},
+		"queue-new-object-iscsidisk-conflict": {
+			pod:          podISCSIDiskDisk,
+			oldObj:       podISCSIDiskConflicts,
+			existingPods: []*v1.Pod{},
+			existingPVC:  &v1.PersistentVolumeClaim{},
+			expectedHint: framework.Queue,
+			expectedErr:  false,
+		},
+		"skip-new-object-iscsidisk-no-conflict": {
+			pod:          podISCSIDiskDisk,
+			oldObj:       podISCSIDiskNoConflicts,
+			existingPods: []*v1.Pod{},
+			existingPVC:  &v1.PersistentVolumeClaim{},
+			expectedHint: framework.QueueSkip,
+			expectedErr:  false,
+		},
+	}
+
+	for name, tc := range testcases {
+		t.Run(name, func(t *testing.T) {
+			logger, _ := ktesting.NewTestContext(t)
+			ctx, cancel := context.WithCancel(context.Background())
+			defer cancel()
+			p := newPluginWithListers(ctx, t, tc.existingPods, nil, []*v1.PersistentVolumeClaim{tc.existingPVC})
+
+			actualHint, err := p.(*VolumeRestrictions).isSchedulableAfterPodDeleted(logger, tc.pod, tc.oldObj, nil)
+			if tc.expectedErr {
+				require.Error(t, err)
+				return
+			}
+
+			require.NoError(t, err)
+			require.Equal(t, tc.expectedHint, actualHint)
```

Comment:
Can you eliminate the usage of `require` package from the test?

> Avoid creating or using assertion libraries. Use standard t.Error or t.Fatal, as necessary.
https://github.com/kubernetes/community/blob/master/sig-scheduling/CONTRIBUTING.md#technical-and-style-guidelines